src/features/appearance/index.ts

```diff
@@ -180,12 +181,31 @@ function generateServiceRibbonWidthStyle(
   // Due to the lowest values for SIDEBAR_WIDTH and ICON_SIZES, this can be computed to a negative value
   const minimumAdjustedIconSize = Math.max(width / 2 + iconSize, 2);
 
+  let sidebarServicesAlignment;
+  switch (sidebarServicesLocation) {
+    case 0:
```

Comment:
Instead of using magic numbers that correspond to similar numbers in the `config.ts`, why not assign to a constant and then use that constant in both places?